doc/guides/networking/overview.rst

```diff
@@ -78,10 +78,10 @@ can be disabled if not needed.
   applications are implemented.
 
 * **LWM2M** OMA Lightweight Machine-to-Machine Protocol
-  (`LwM2M specification 1.0.2`_) is
-  supported via the "Register Device" API (Register, De-Register and Update)
-  and has template implementations for Security, Server, Device Management and
-  Firmware objects. Bootstrap support is currently not supported.
+  (`LwM2M specification 1.0.2`_) is supported via the "Bootstrap", "Client
+  Registration", "Device Management & Service Enablement" and "Information
+  Reporting" interfaces.  The required core of the LwM2M objects are implemented
```

Comment:
core _is_ implemented, perhaps?

Reply:
Changed to:
The required core LwM2M objects are implemented as well as several IPSO Smart Objects.